### PR TITLE
Add photo to notes

### PR DESCRIPTION
**Pull request by jasonbrandt42**

### CODE REVIEW

File: src/app/dashboard/components/item-request/item-request.component.ts [hunk 1]
```diff
@@ -17,20 +19,23 @@ import { Status } from 'src/app/core/constants/enums';
...
+  imagename: string = '';
   currentNoteVal: string;
   orderNoteFC: FormControl = new FormControl('');
   orderNoteFC$: Observable<string>;
 
-  constructor() {}
+  constructor(private userService: UserService) { }
 
   ngOnInit() {
+    this.userProfile = this.userService.getCurrentUserProfile();
```
Comment (johnmcguin) [inline src/app/dashboard/components/item-request/item-request.component.ts:37]: These components are built with push based vs. pull based architecture and the preference would be to have the service fetch userprofile and expose it to child components via the state observable as opposed to the child component fetching it.

File: src/app/dashboard/components/item-share/item-share.component.ts [hunk 2]
```diff
@@ -33,9 +38,11 @@ export class ItemShareComponent implements OnInit, OnDestroy {
   orderNoteFC: FormControl = new FormControl('');
   orderNoteFC$: Observable<string>;
 
-  constructor(private dialog: MatDialog, private router: Router) {}
+  constructor(private dialog: MatDialog, private router: Router, private userService: UserService) { }
 
   ngOnInit() {
+    this.userProfile = this.userService.getCurrentUserProfile();
```
Comment (johnmcguin) [inline src/app/dashboard/components/item-share/item-share.component.ts:44]: same comment from request component

File: src/app/dashboard/components/item-image-canvass/item-image-canvass.component.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { OnInit, Renderer2, Component, ElementRef, ViewChild, Input } from '@angular/core';
```
Comment (johnmcguin) [inline src/app/dashboard/components/item-image-canvass/item-image-canvass.component.ts:1]: just a filename typo quick fix canvass vs canvas

**Comment by jasonbrandt42** [comment]
I have updated linting and fixed spelling errors this is ready for merge

**Pull request merged**